packages/core/src/utils.ts

```diff
@@ -123,8 +123,9 @@ export async function loadPlugins(context: IContext): Promise<IPluginsLoaded> {
  * @param context Global context object for Octorelease
  */
 export async function verifyConditions(context: IContext): Promise<void> {
-    context.version.new = Inputs.newVersion || context.version.new;
```

Comment:
https://github.com/zowe-actions/octorelease/blob/611d26ffc8dad793bd254a1c66ba41e120844efa/packages/core/src/inputs.ts#L68-L70

Is this where the `undefined` was coming from?